subsys/bluetooth/host/Kconfig.gatt

```diff
@@ -125,6 +125,21 @@ config BT_GATT_NOTIFY_MULTIPLE
 	  This option enables support for the GATT Notify Multiple
 	  Characteristic Values procedure.
 
+if BT_GATT_NOTIFY_MULTIPLE
+
+config BT_GATT_NOTIFY_MULTIPLE_FLUSH_MS
+	int "Delay for batching multiple notifications in a single PDU."
+	default 1
+	range 0 4000
+	help
+	  Sets the time (in milliseconds) during which consecutive GATT
+	  notifications will be tentatively appended to form a single
+	  ATT_MULTIPLE_HANDLE_VALUE_NTF PDU.
+	  If set to 0, batching is disabled.
```

Comment:
And in that case the only way to send multi-notification is by using the `bt_gatt_notify_multiple` function. Should that be mentioned here?

Reply:
yes, will do